test/Feature/JsonOperatorSpec.hs

```diff
@@ -26,7 +26,7 @@ spec actualPgVersion = describe "json and jsonb operators" $ do
 
     it "fails on bad casting (data of the wrong format)" $
       get "/complex_items?select=settings->foo->>bar::integer"
-        `shouldRespondWith` [json| {"hint":null,"details":null,"code":"22P02","message":"invalid input syntax for integer: \"baz\""} |]
+        `shouldRespondWith` [json| {"hint":null,"details":null,"code":"22P02","message":"invalid input syntax for type integer: \"baz\""} |]
```

Comment:
What PostgreSQL version are you using? pg 12? If so you could conditionally validate the output according to the version. There's a similar logic below:

https://github.com/PostgREST/postgrest/blob/9b1224827a4c4b71ffe924e54f024b08df93fecf/test/Feature/JsonOperatorSpec.hs#L54-L64 

This would make the test runs go green.

Reply:
You'll need to add pg12 version in https://github.com/PostgREST/postgrest/blob/9b1224827a4c4b71ffe924e54f024b08df93fecf/src/PostgREST/Types.hs#L506-L513

Reply:
Yes, I'm running version 12.1 adjusted the test case accordingly, thanks for pointing this out. I just saw it being red and fixed it according to the expected output. I assumed it to be a typo.